index.html

```diff
@@ -21,18 +21,27 @@
   <div class="main-container">
       <header class="navbar">
           <a href="" class="logo">Kinyera Amos</a>
-          <nav class="mobile_menu">
-              <div class="menu-btn">
-                  <div class="humbergure"></div>
-              </div>
-          </nav>
-          <nav class="desktop-menu">
-            <ul class="nav-link">
-                  <li><a href="#" class="link">Portfolio</a></li>
-                  <li><a href="#" class="link">About</a></li>
-                  <li><a href="#" class="link">Contact</a></li>
+          <div id="myNav" class="overlay">
+              <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+              <ul class="overlay-content">
+                 <li><a href="#" >Portfolio<i class="fa fa-angle-right"></i></a></li>
+                  <li><a href="#">About<i class="fa fa-angle-right"></i></a></li>
+                  <li><a href="#">Contact<i class="fa fa-angle-right"></i></a></li>
```

Comment:
- [ ] Kindly decrease the spacing between menu items and make the close button smaller to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/167429801-a49d6b1c-cb53-4e1e-a0fa-625a07d4e6a3.png) | ![image](https://user-images.githubusercontent.com/24703171/167429899-fa86c196-c1f0-4fd1-bc98-864e20ed919e.png) |

- [ ] Please make the menu links `Portfolio`, `About` and `Contact` when pressed go to the right part of the page and the menu disappears.

Reply:
- Hi, failed to make the mobile menu disappear when you click the link, secondly, it's not among the requirements for the project.